Core/Services/Auth/RegisterService.cs

```diff
@@ -13,15 +13,13 @@
 
 namespace Core.Services.Auth
 {
-    class RegisterService : IRegisterService
+    class RegisterService : AuthServicesProvider, IRegisterService
```

Comment:
Is the default internal access modifier correct for this class?

Reply:
Yes, because I use this class only in this folder